10_install_server/12_install_root_server_rhel.txt

```diff
@@ -0,0 +1,99 @@
+=== Install Rudder Root server on Enterprise Linux ( RedHat or RedHat-like )
+
+==== Add the Rudder packages repository
+
+Run the following command as root:
+
+Note: <version> is your RHEL version
```

Comment:
I seriously think that giving copypastable code bits is a Quick start guide role, the documentation is meant to be read and understood not blindly copypasted.

Also, I have faith in the fact that our users are skilled enought to paste something in a notepad or something to be modified before copying it to 100 machines after having evaluated Rudder one one machine.